### PR TITLE
Adding contract role to job board

### PR DESCRIPTION
**Pull request by ziyenl**
Adding contract role to job board

### CODE REVIEW

**Comment by ziyenl** [comment]
Adding contract role to job board

File: jobs/partners-capital-senior-full-stack-contract-software-developer.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+url: https://www.partners-cap.com/
+location: London, England
+          
+contract: contract
+contact:
+    name: Suzi Huntington
+    email: suzi.huntington@partners-cap.com
+    phone: +44 (0)20 7938 5228
+    # <additional contact fields>: <additional value>
+created: !!timestamp 2018-04-25
```
Comment (lordmauve) [inline jobs/partners-capital-senior-full-stack-contract-software-developer.html]: This should be set to today's date.

**Review by lordmauve — COMMENTED** [review]
Otherwise, looks correct to me.

**Comment by ziyenl** [comment]
I have set it to today's date.

**Comment by stestagg** [comment]
Hi

Thanks for submitting this job!

Unfortunately, you have to submit the requests to this repository: https://github.com/pythonjobs/jobs 

If we merge this here, it’ll appear on the site briefly (in a. Weird format) but be overwritten by any subsequent changes.

I’ll close this PR so it doesn’t get merged by accident, but you should be able to raise an identical pr against the linked repo.

Thanks 

Steve

**Pull request closed without merging**